bin/express

```diff
@@ -398,18 +448,20 @@ function main() {
   var appName = createAppName(path.resolve(destinationPath)) || 'hello-world'
 
   // View engine
-  if (program.view === undefined) {
+  if (program.view) {
```

Comment:
I could see both ways, I guess. So this PR would need to preserve the existing behavior. Blowing up may be good, so feel free to make a separate PR for that change, so the rest of the members can weight in on that change :) !